### PR TITLE
Remove Wraith check

### PR DESCRIPTION
**Pull request by wildlyinaccurate**
Hardly anybody runs this check -- not even the testers use Wraith that much these days.

I think having stuff like this on the checklist just encourages people to tick all the boxes just to get their PR through, even if they haven't run everything.

### CODE REVIEW

**Comment by Integralist** [comment]
Makes sense, and I agree if this is what's happening. I guess the bigger and more important question beyond this PR is: "why are testers not using Wraith? is there something else they're using in its place? and why the change?"

**Comment by wildlyinaccurate** [comment]
From what I understand, it's still used for regression testing. It's just not really necessary for the vast majority of PRs, and for the ones that it *is* necessary, most devs don't run it anyway.

**Comment by dgm34** [comment]
Was there ever a decision about this? Anecdotally, it seems like no dev uses wraith and - to a good approximation - this box is never ticked on Responsive News PRs. It's more something that the test team will run themselves (or suggest to us that we run it) as part of their testing process.

I say merge it!

**Comment by staylos92** [comment]
I agree with the above. Running Wraith generally isn't a dev task and is down to us testers to run as part of regression or when testing certain PRs.

File: responsive-news.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 - [ ] Milestone added
 - [ ] Pull request URL added to Jira ticket
 
-######
```
Comment (samfrench) [inline responsive-news.md]: Don't we need the additional spaces, or the list won't show correctly?

Comment (wildlyinaccurate) [inline responsive-news.md]: It looks fine without: https://github.com/wildlyinaccurate/pr-templates/blob/remove-wraith-check/responsive-news.md

Comment (dgm34) [inline responsive-news.md]: My Sublime plugin removes the space automatically, but I think it renders fine. If it's broken when we merge i'll fix in a different text editor and merge it straight in!

**Pull request merged**